### DISCUSSION TITLE
How to set up multiple identical characters?

### DISCUSSION THREAD

**Discussion by deux2303** [isid0re/SoloLeveling#168] [Q&A]
I designated my profile as SCL-PAL, SCL-PAL02, SCL-PAL03, in order to raise several paladins. However, an error has been detected.
![갑자기 에러](https://user-images.githubusercontent.com/79701816/111190512-7ff4cd00-85fa-11eb-8187-2991f25b021d.jpg)
The same error occurred when generating with different types of character. Also, the same error occurred when multiple programs were run with one character each. And There was an error, but they still started leveling. But they often died without taking potions. I am not good in English. I hope the meaning of this question has been conveyed. I am using the program well. Thank you very much for make this.

**Comment by isid0re** [comment]
Please confirm you are using the https://www.github.com/blizzhackers/kolbot repo and not a modified repo from another source. Also are you playing on a modded d2.

**Reply by deux2303** [reply]
Thank you for your answer. I used that. I solved this problem an hour ago. The name of my folder was too long. It was an error caused by that. The folder has been renamed and resolved.
